person.rb

```diff
@@ -20,6 +26,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
```

Comment:
- [ ] Great job. here kindly ensure the add rental method takes in the book and date. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. something like 👇🏾 

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
Changes done. Thank you for the suggestions @tufoinnkuo10 .